### ISSUE TITLE
Default check keeps failing

### ISSUE THREAD

**Issue by Flink** [progrium/dokku#1248]
Hi!

On one of my server (ubuntu 14.04, dokku 0.3.19) when the default check is made, it keeps failing no matter what. And since other process than `web` don’t use the CHECKS file, it is more than annoying.

I don’t know why it works but I’ve found that removing the `-q` flag from grep at https://github.com/progrium/dokku/blob/master/plugins/checks/check-deploy#L94 will make the check pass every time.

**Comment by michaelshobbs** [comment]
The default check is just making sure the new container is running. There was a discussion at some point about bypassing the checks altogether via an environment variable. We'd definitely welcome a PR that does this.

**Comment by josegonzalez** [comment]
```
-q, --quiet, --silent
              Quiet; do not write anything to standard output.  Exit immediately with zero status if any match is found, even if an error was detected.  Also see the -s or --no-messages option.  (-q is specified by POSIX.)
```

**Comment by Flink** [comment]
Yes I don’t really understand why it fails since it should have an exit code of 0. And manually in a console it seems to work properly.

**Comment by michaelshobbs** [comment]
It's very possible that the flag change is unrelated....

**Comment by Flink** [comment]
Well it was failing almost every time when doing the check and after removing the flag it works every time. And I did a lot of deploy/rebuild.

**Comment by Flink** [comment]
@michaelshobbs about your comment on bypassing the checks, I just opened this PR: https://github.com/progrium/dokku/pull/1261 :grin:

**Comment by michaelshobbs** [comment]
@Flink excellent! thank you

**Comment by michaelshobbs** [comment]
Fixed in #1261
